code/edge/src/androidTest/java/com/adobe/marketing/mobile/ConsentStatusChangeFunctionalTests.java

```diff
@@ -12,7 +12,7 @@
 package com.adobe.marketing.mobile;
 
 import static com.adobe.marketing.mobile.services.HttpMethod.POST;
-import static com.adobe.marketing.mobile.util.FunctionalTestHelper.*;
+import static com.adobe.marketing.mobile.util.TestHelper.*;
```

Comment:
nit: not really part of this "move" PR, but can you expand the imports so it's not using the wildcard?